tools/run_tests/dockerize/build_and_run_docker.sh

```diff
@@ -58,7 +58,13 @@ docker run \
   -e "KOKORO_BUILD_NUMBER=$KOKORO_BUILD_NUMBER" \
   -e "KOKORO_BUILD_URL=$KOKORO_BUILD_URL" \
   -e "KOKORO_JOB_NAME=$KOKORO_JOB_NAME" \
+  -e "KOKORO_ARTIFACTS_DIR=$KOKORO_ARTIFACTS_DIR" \
+  -e "GIT_ORIGIN_URL=$(git -C $git_root remote get-url origin)" \
```

Comment:
(totally optional) At this point we potentially could pass the last property as an env var too, to avoid `subprocess` calls whatsoever
```suggestion
  -e "GIT_ORIGIN_URL=$(git -C $git_root remote get-url origin)" \
  -e "GIT_COMMIT_SHORT=$(git -C $git_root rev-parse --short HEAD)" \
```

Reply:
Done.